### PR TITLE
Added documentation for Selection and working-with-treeview

### PR DESCRIPTION
**Pull request by SuthiYuvaraj**
Added Sample for Selection and Working With TreeView UG

### CODE REVIEW

File: MAUI/TreeView/working-with-treeview.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Working with TreeView in .NET MAUI TreeView control | Syncfusion
+description: Learn here all about Working with TreeView support in Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
```
Comment (M-A32) [inline MAUI/TreeView/working-with-treeview.md:4]: description: Learn here about Working with TreeView support in Syncfusion .NET MAUI TreeView (SfTreeView) control and more.

File: MAUI/TreeView/working-with-treeview.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+The `Loaded` event is used for the following use case:
+
+* To scroll the desired item by using the `BringIntoView`.
+
+### Tapped event
+
+The `ItemTapped` event will be triggered whenever tapping the item.  `ItemTappedEventArgs` has the following members which provides the information for `ItemTapped` event:
```
Comment (M-A32) [inline MAUI/TreeView/working-with-treeview.md:35]: The `ItemTapped` event will be triggered whenever tapping the item.  `ItemTappedEventArgs` has the following members which provide the information for the `ItemTapped` event:

File: MAUI/TreeView/selection.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+The TreeView allows changing the selection foreground color for the selected items by using the `SelectionForegroundColor` property. You can also change the selection foreground color at runtime.
+
+N> `SelectionForegroundColor` is applicable only for unbound mode.
+
+## Events
+
+### SelectionChanging Event
+
+The `SelectionChanging` event is raised while selecting an item at the execution time. The `ItemSelectionChangingEventArgs` has the following members which provides the information for `SelectionChanging` event:
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:110]: The `SelectionChanging` event is raised while selecting an item at the execution time. The `ItemSelectionChangingEventArgs` has the following members which provide the information for `SelectionChanging` event:

File: MAUI/TreeView/selection.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+# Selection in .NET MAUI TreeView (SfTreeView)
+
+This section explains how to perform selection and its related operations in the TreeView.
+
+## UI Selection
+The TreeView allows selecting the items either programmatically or touch interactions by setting the `SelectionMode` property value to other than `None`. The control has different selection modes to perform selection operations as listed as follows.
+
+* `None`: Allows disabling the selection.
+* `Single`: Allows selecting the single item only. When clicking on the selected item, selection not will not be cleared. This is the default value for `SelectionMode`.
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:18]: * `Single`: Allows selecting the single item only. When clicking on the selected item, the selection will not be cleared. This is the default value for `SelectionMode`.

File: MAUI/TreeView/selection.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+* `SingleDeselect`: Allows selecting the single item only. When clicking on the selected item, selection gets cleared.
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:19]: * `SingleDeselect`: Allows selecting the single item only. When clicking on the selected item, the selection gets cleared.

File: MAUI/TreeView/selection.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+* `Multiple`: Allows selecting more than one item. Selection is not cleared when selecting more than one items. When clicking on the selected item, selection gets cleared.
+* `Extended`: Allows to select the multiple items using the common key modifiers.
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:21]: `Extended`: Allows to select multiple items using the common key modifiers.

File: MAUI/TreeView/selection.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+{% tabs %}
+{% highlight c# %}
+treeView.SelectedItems.Clear();
+{% endhighlight %}
+{% endtabs %}
+ 
+### CurrentItem vs SelectedItem
+
+The TreeView gets the selected item by using the `SelectedItem` properties. Both `SelectedItem` and `CurrentItem` returns the same data object when selecting single item. When selecting more than one item, the `SelectedItem` property returns the first selected item, and the `CurrentItem` property returns the last selected item.
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:77]: The TreeView gets the selected item by using the `SelectedItem` properties. Both `SelectedItem` and `CurrentItem` return the same data object when selecting a single item. When selecting more than one item, the `SelectedItem` property returns the first selected item, and the `CurrentItem` property returns the last selected item.

File: MAUI/TreeView/selection.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+
+The TreeView allows to select the items through keyboard interactions. Behavior of key navigation is explained as follows:
+
+* When the `SelectionMode` is `Single` or `SingleDeselect`, the selected item is highlighted with `FocusBorderColor` around the item while key navigation.
+
+* When the `SelectionMode` is `Multiple` or `Extended`, the `FocusBorderColor` will set to the `CurrentItem`.
+
+## FocusBorderColor
+
+The `FocusBorderColor` property is used to set the border color for the current focused item.The default color is `Color.FromArgb("#000000")`.
```
Comment (M-A32) [inline MAUI/TreeView/selection.md:159]: The `FocusBorderColor` property is used to set the border color for the current focused item. The default color is `Color.FromArgb("#000000")`.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

**Comment by Lakshminatarajan** [comment]
Include file path in the toc file

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**